doc/articles/how-uno-works.md

```diff
@@ -4,34 +4,33 @@ How does Uno Platform make the same application code run on all platforms?
 
 #### On Windows
 
-On Windows (the UWP head project), an Uno Platform application isn't actually using Uno.UI at all. It's compiled just like a single-platform UWP (or WinUI 3) application, using Microsoft's tooling.
+On Windows (the Windows head project), an Uno Platform application isn't using Uno.UI at all. It's compiled just like a single-platform WinUI (or UWP) application, using Microsoft's own tooling.
 
 The rest of this article discusses how the Uno.UI tooling allows UWP/WinUI-compatible XAML and C# applications to run on non-Windows platforms.
 
 ## Uno.UI at runtime
 
-The [`Uno.UI` library](https://www.nuget.org/packages/Uno.UI/) completely reproduces the UWP ([or WinUI 3](updating-to-winui3.md)) API surface: all namespaces (`Windows.UI.Xaml`, `Windows.Foundation`, `Windows.Storage`, etc), all classes, all class members. Insofar as possible, the same look and behavior as on Windows is replicated on all other platforms.
+The [`Uno.UI` library](https://www.nuget.org/packages/Uno.UI/) completely reproduces the WinUI (or UWP) API surface: all namespaces (`Microsoft.UI.Xaml`, `Windows.Foundation`, `Windows.Storage`, etc), all classes, all class members. Insofar as possible, the same look and behavior as on Windows is replicated on all other platforms.
 
 Note that, as the API surface is very large, some parts of it are included but not implemented. These features are marked with the `Uno.NotImplementedAttribute` attribute, and a code analyzer included with the Uno.UI package will generate a warning for any such features that are referenced. You can see a complete list of supported APIs [here](implemented-views.md).
 
 ### How the UI is rendered
 
-As an application developer, you normally don't need to worry about exactly how Uno.UI renders your visual tree to the screen on each platform. For every platform, the public API -
- panels, control templates, measurement and arranging, etc - is the same. However, it's sometimes useful to know what's going on at render-time, for example if you're [intermixing native views with XAML views](native-views.md) on a platform-specific basis.
+As an application developer, you normally don't need to worry about exactly how Uno.UI renders your visual tree to the screen on each platform. For every platform, the public API - panels, control templates, measurement and arranging, etc - is the same. However, it's sometimes useful to know what's going on at render-time, for example if you're [intermixing native views with XAML views](native-views.md) on a platform-specific basis.
 
 #### Web (WebAssembly)
 
 On the web, each XAML element is converted into an appropriate [HTML element](https://developer.mozilla.org/en-US/docs/Glossary/Element). Panels, controls, and other 'intermediate' elements in the visual tree are converted to `<div/>` elements, whereas 'leaf' elements like `TextBlock`, `Image` etc get converted into more specific tags (`<p/>`, `<img/>` etc).
 
-#### iOS
+#### iOS and mac Catalyst
```

Comment:
Mac Catalyst?
mac Catalyst?

They both feel weird

Reply:
Based on Apple's docs, it's `Mac Catalyst`, so we can change it everywhere :) I'll take note. Thanks!